### ISSUE TITLE
[Composer] [ErrorException] Undefined index: versions

### ISSUE THREAD

**Issue by jonathaningram** [zendframework/ZF2Package#15]
I think that packages.zendframework.com might have a broken setting for Composer?

Here's the error if it helps:

```
$ php composer.phar update -v

  [ErrorException]           
  Undefined index: versions  

Exception trace:
 () at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Repository/ComposerRepository.php:120
 Composer\Util\ErrorHandler::handle() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Repository/ComposerRepository.php:120
 Composer\Repository\ComposerRepository->loadRepository() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Repository/ComposerRepository.php:112
 Composer\Repository\ComposerRepository->initialize() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Repository/ArrayRepository.php:144
 Composer\Repository\ArrayRepository->getPackages() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/DependencyResolver/Pool.php:66
 Composer\DependencyResolver\Pool->addRepository() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Installer.php:247
 Composer\Installer->doInstall() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Installer.php:178
 Composer\Installer->run() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Command/UpdateCommand.php:70
 Composer\Command\UpdateCommand->execute() at phar:///Users/jonathaningram/example.com/composer.phar/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:239
 Symfony\Component\Console\Command\Command->run() at phar:///Users/jonathaningram/example.com/composer.phar/vendor/symfony/console/Symfony/Component/Console/Application.php:193
 Symfony\Component\Console\Application->doRun() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Console/Application.php:86
 Composer\Console\Application->doRun() at phar:///Users/jonathaningram/example.com/composer.phar/vendor/symfony/console/Symfony/Component/Console/Application.php:106
 Symfony\Component\Console\Application->run() at phar:///Users/jonathaningram/example.com/composer.phar/src/Composer/Console/Application.php:66
 Composer\Console\Application->run() at phar:///Users/jonathaningram/example.com/composer.phar/bin/composer:13
 require() at /Users/jonathaningram/example.com/composer.phar:15

update [--prefer-source] [--dry-run] [--dev] [--no-scripts] [packages1] ... [packagesN]
```

In my `composer.json` I had:

``` json
        {
            "type": "composer",
            "url": "http://packages.zendframework.com/"
        },
```

I've had to remove it to fix the error so I am assuming there is something misconfigured on packages.zendframework.com?

**Comment by weierophinney** [comment]
Yeah, there was a snafu in our composer repo build. Working on a fix now.

**Comment by weierophinney** [comment]
Fixed as of faaed893c7
